vulnerabilities/templates/packages.html

```diff
@@ -18,6 +18,14 @@
                 <div>
                     {{ page_obj.paginator.count|intcomma }} results
                 </div>
+                <form method="get" style="display: inline;">
+                    {% if search %}<input type="hidden" name="search" value="{{ search }}">{% endif %}
+                    <select name="vulnerable_only" class="select" id="vulnerable-select" onchange="this.form.submit()">
+                        <option value="">All Packages</option>
+                        <option value="true" {% if request.GET.vulnerable_only == 'true' %}selected{% endif %}>Vulnerable Only</option>
+                        <option value="false" {% if request.GET.vulnerable_only == 'false' %}selected{% endif %}>Non-Vulnerable Only</option>
+                    </select>
+                </form>
```

Comment:
I'm not sure if this is how we want to have filters, how about a filter icon to the left with drop-down with All, Non-vulnerable, Vulnerable options.